client/Components/Header/Header.jsx

```diff
@@ -79,17 +102,46 @@ const Header = ({ loggedIn, setLoggedIn }) => {
             <Link to="/">
               <Thumbnail src="https://i.imgur.com/XYg49nh.jpg"></Thumbnail>
             </Link>
-            <Link to="/">
-              <AvatarWrapper src={avatar}></AvatarWrapper>
-            </Link>
-            <Dropdown value="selection" onChange={(e) => handleChange(e)}>
-              <option key={userOptions[0].value} value={userOptions[0].value}>
+            {/* <Link to="/"> */}
+            <div
+              style={{
+                display: "flex",
+                alignItem: "center",
+              }}
+            >
+              <AvatarWrapper src={loggedIn.userAvatar}></AvatarWrapper>
+              {/* </Link> */}
+              <Dropdown value="selection" onChange={(e) => handleChange(e)}>
+                {userOptions.map((option, index) => {
+                  if (!index) {
+                    return (
+                      <option
+                        key={option.value}
+                        value={option.value}
+                        defaultValue
+                      >
+                        {option.label}
+                      </option>
+                    );
+                  } else {
+                    return (
+                      <option key={option.value} value={option.value}>
+                        {option.label}
+                      </option>
+                    );
+                  }
+                })}
+                {/* <option key={userOptions[0].value} value={userOptions[0].value}>
                 {userOptions[0].label}
               </option>
               <option key={userOptions[1].value} value={userOptions[1].value}>
                 {userOptions[1].label}
               </option>
-            </Dropdown>
+              <option key={userOptions[2].value} value={userOptions[2].value}>
+                {userOptions[1].label}
+              </option> */}
+              </Dropdown>
+            </div>
```

Comment:
It's good practice to only have working code in the main branch. Let's get rid of any unused code here and console logs as well.